File: app/src/main/java/com/commit451/gitlab/model/api/Runner.java
package com.commit451.gitlab.model.api;

import com.squareup.moshi.Json;

import org.parceler.Parcel;

/**
 * A runner. It runs builds. yeah
 */
@Parcel
public class Runner {

    @Json(name = "id")
    long id;
    @Json(name = "description")
    String description;
    @Json(name = "active")
    boolean active;
    @Json(name = "is_shared")
    boolean isShared;
    @Json(name = "name")
    String name;

    public long getId() {
        return id;
    }

    public String getDescription() {
        return description;
    }

    public boolean isActive() {
        return active;
    }

    public boolean isShared() {
        return isShared;
    }

    public String getName() {
        return name;
    }
}

File: app/src/main/java/com/commit451/gitlab/model/api/Runner.kt
package com.commit451.gitlab.model.api

import android.os.Parcelable
import com.squareup.moshi.Json
import kotlinx.android.parcel.Parcelize

/**
 * A runner. It runs builds. yeah
 */
@Parcelize
data class Runner(
    @Json(name = "id")
    var id: Long = 0,
    @Json(name = "description")
    var description: String? = null,
    @Json(name = "active")
    var isActive: Boolean = false,
    @Json(name = "is_shared")
    var isShared: Boolean = false,
    @Json(name = "name")
    var name: String? = null
) : Parcelable
